<launch>
  <!-- Start Gazebo simulation with your robot model and camera -->
   <!-- <include file="$(find xarm_gazebo)/launch/xarm6_beside_table.launch" /> -->
  
  <!-- Load the robot model and camera -->
  <!-- Include other launch files or spawn the robot using Gazebo ROS plugins -->

  <!-- Start your visual servoing node -->
  <node name="visual_servo_node" pkg="visual_servo" type="visual_servo_node.py">
    <param name="target_x" value="320" />  <!-- Adjust the target_x and target_y as needed -->
    <param name="target_y" value="240" />
    <param name="p_gain" value="0.001" />
  </node>
    <node name="perception" pkg="visual_servo" type="perception.py">

  </node>
 <include file="$(find realsense2_camera)/launch/rs_camera.launch"></include>
</launch>
